src/images/florence.svg

```diff
@@ -1 +1 @@
.1.7 6.3 1.1 9.5 1.1 26.6 0 48-25.2 48-56.3z"/><path class="st5" d="M123.4 33.3c0-3 .2-6 .6-8.9 3.2-.8 6.4-1.1 9.8-1.1C161 23.3 183 49.1 183 81c0 3-.2 6-.6 8.9-3.2.8-6.4 1.1-9.8 1.1-27.2.1-49.2-25.7-49.2-57.7z"/><circle class="st0" cx="116.3" cy="99.3" r="6.3"/><path class="st0" d="M110.4 97.9l-2-4.7 6.1 1.5-4.1 3.2z"/><circle class="st0" cx="149.5" cy="99.3" r="6.3"/><path class="st0" d="M155.4 97.9l2-4.7-6.1 1.5 4.1 3.2z"/><path class="st5" d="M161.2 51.9c3.9 0 7.7.9 11 2.6.3.2.6-.3.3-.5-4.1-3.3-9.4-5.3-15.1-5.3-13.3 0-24.2 10.8-24.2 24.1 0 7.3 3.3 13.9 8.5 18.3.3.2.6-.1.4-.4-3.2-4.1-5-9.2-5-14.7-.1-13.3 10.7-24.1 24.1-24.1zm-40.3-4c3.7 1.3 6.9 3.4 9.5 6 .2.3.6-.1.5-.4-2.8-4.5-7.2-8.1-12.6-9.9-12.6-4.4-26.4 2.3-30.7 14.9-2.4 6.9-1.4 14.2 2 20.1.2.3.6.1.5-.2-1.7-4.9-1.8-10.3 0-15.6 4.4-12.6 18.1-19.3 30.8-14.9z"/><path class="st5" d="M114 65.8c3.9.6 7.4 2.1 10.5 4.2.3.2.6-.2.4-.4-3.6-3.9-8.5-6.6-14.2-7.4-13.2-2-25.5 7.1-27.4 20.3-1.1 7.2.2 14.6 4.7 19.7.2.3.7 0 .5-.3-2.5-4.5-1.7-10.2-.8-15.7C89.5 73 100.8 63.9 114 65.8z"/><path class="st0" d="M126 123.8l-9.8-4.5c-1.6-1.2-1.5-3.3.6-3.3 0 0 10.5 1.3 16.2 1.6 5.7.2 14.9-.8 14.9-.8 2 0 1.9 2.1.5 3.3l-7.4 3.6c-4.4 2.4-10.1 2.3-15 .1z"/><path d="M131.5 112.9l-2.8-1.3c-.4-.4-.4-.9.2-.9 0 0 3 .4 4.6.4 1.6.1 4.2-.2 4.2-.2.6 0 .5.6.1.9l-2.1 1c-1.2.8-2.8.7-4.2.1z" fill="#bfb9a5"/><path d="M137.8 58.2c10.4.8 20.4 2.8 29.8 5.9 2.8.9 7.8-7.9 9.6-9.9l-4.1-8.4c2-2.3-72.3 9.7-72.3 9.7 1.4 2.3 3.4 4.5 6.2 4.1 9.9-1.7 20.2-2.2 30.8-1.4z" opacity=".5" fill="#c16c37"/><path class="st14" d="M134 137.4c13.2 0 14.4-.2 2 5.7-12.6 5.9-29.4.5-38.6-8.4-1.8-1.7 6.3-3.5 7.7-5.6 0-.1 15 8.3 28.9 8.3zM149.3 171.8c-10.8 7.5-36.2 11.8-36.2 11.8h54.5v-29.2c.1-.1-7.4 9.9-18.3 17.4z"/><path class="st14" d="M136.4 136.4c-13.2 0-14.4-.2-2 5.7 12.6 5.9 30.4 4.5 39.6-4.4 1.8-1.7-7.3-7.5-8.7-9.6 0-.1-15 8.3-28.9 8.3z"/><path class="st0" d="M139.2 199.9h-7.1c-1.9 0-3.4-1.5-3.4-3.4V172c0-1.9 1.5-3.4 3.4-3.4h7.1c1.9 0 3.4 1.5 3.4 3.4v24.6c0 1.8-1.5 3.3-3.4 3.3z"/><path class="st0" d="M151.3 180.7v7.1c0 1.9-1.5 3.4-3.4 3.4h-24.6c-1.9 0-3.4-1.5-3.4-3.4v-7.1c0-1.9 1.5-3.4 3.4-3.4H148c1.8 0 3.3 1.5 3.3 3.4z"/><path class="st10" d="M134.5 54c9 0 17.8 1.1 26.1 3.1 2.4.6 5-.3 6.4-2.2l7.7-10.5c1.5-2.1 1.2-4.9-.8-6.6-10.2-8.8-24.1-14.2-39.4-14.2-15.5 0-29.5 5.5-39.7 14.4-2 1.7-2.3 4.5-.8 6.6l7.6 10.4c1.4 1.9 4 2.7 6.4 2.2 8.5-2.1 17.3-3.2 26.5-3.2z"/><path class="st14" d="M135.8 54c9 0 17.8 1.1 26.1 3.1 2.4.6 5-.3 6.4-2.2l4.4-5.9c-11-5.1-24.2-6.8-37.6-4.1C125 47 116 51.3 108.6 57.2c.3 0 .5-.1.8-.1 8.4-2 17.3-3.1 26.4-3.1z"/><path class="st0" d="M134.8 30.8c15.1 0 29.4 5.1 39.8 13.5l.8-1c1.6-2.1 1.2-5-.8-6.8-10.5-9-24-14.5-39.7-14.5-15.8 0-30.8 5.6-41.3 14.8-2.1 1.8-2.4 4.6-.8 6.8l1.8 2.5c.1-.1.2-.3.4-.4 10.4-9.3 24-14.9 39.8-14.9z"/><path d="M127.7 124.3c4.5 1.6 9.5 1.5 13.5-.7l.2-.1c-1.6-.8-3.8-1.2-6.2-1.2-3.1 0-5.8.8-7.5 2z" fill="#e75f67"/><path class="st14" d="M197.8 279.3c0 1.9-1.3 3.4-3 3.4h-52.7c26.3-10.7 42.8-17.2 54.3-34l1.4 30.6z"/></svg>
```

Comment:
Probably don't need this image anymore?

Reply:
The Florence image gets called in the footer as well, so I had to keep it!